src/classes/HH_OppContactRoles_TDTM.cls

```diff
@@ -143,7 +155,7 @@ public class HH_OppContactRoles_TDTM extends TDTM_Runnable {
             
             // query for OppConRoles where primary=true and OppID in arg set
             // query changed to also include Role = default role, because Contact Merge scenarios were clearing isPrimary (not by our code)!
-            map<String,Id> contactOppToContactRole = new map<String,Id>();
+            Map<String,Id> contactOppToContactRole = new Map<String,Id>();
```

Comment:
Since we fixed the spacing on the other maps, just going to mention the space missing here but up to you if you want to fix it.